crates/astria-sequencer/local.env.example

```diff
@@ -0,0 +1,11 @@
+# Log level for the sequencer
+ASTRIA_COMPOSER_LOG="astria_composer=info"
+
+# Address of the API server
+ASTRIA_SEQUENCER_GENESIS_FILE="test-genesis.json"
+
+# Address of the RPC server for the sequencer chain
+ASTRIA_SEQUENCER_LISTEN_ADDR="http://127.0.0.1:26658"
+
+# Path to rocksdb
+ASTRIA_SEQUENCER_DB_FILEPATH="../"
```

Comment:
if this is supposed to be a directory (which I think it should be) rename this to `ASTRIA_SEQUENCER_DB_DATADIR`?